import-service/lambda-functions/importFileParser.ts

```diff
@@ -41,10 +44,15 @@ export async function handler(event: S3Event) {
 
     if (res.Body instanceof stream.Readable) {
       res.Body.pipe(csv())
-        .on("data", (data: { [key: string]: string }) => parsed.push(data))
-        .on("end", () => {
-          console.log("Parsed CSV:", parsed);
-        });
+        .on("data", async (data: { [key: string]: string }) => {
+          parsed.push(data);
+          const command = new SendMessageCommand({
+            QueueUrl: SQS_URL,
+            MessageBody: JSON.stringify(data),
+          });
+          await sqsClient.send(command);
```

Comment:
Products parsing is blocked with sending a message to SQS with each chunk.
It would be better to separate products parsing and sending messages to SQS.